source/modules/MEI.shared.xml

```diff
@@ -7454,26 +7454,38 @@
       <rng:zeroOrMore>
         <rng:ref name="model.headLike"/>
       </rng:zeroOrMore>
+      <rng:choice>
+        <rng:group>
+          <rng:oneOrMore>
+            <rng:oneOrMore>
+              <rng:ref name="resp"/>
+            </rng:oneOrMore>
+            <rng:oneOrMore>
+              <rng:choice>
+                <rng:ref name="model.nameLike.agent"/>
+                <rng:ref name="name"/>
+              </rng:choice>
+            </rng:oneOrMore>
+          </rng:oneOrMore>
+        </rng:group>
+        <rng:group>
+          <rng:oneOrMore>
+            <rng:oneOrMore>
+              <rng:choice>
+                <rng:ref name="model.nameLike.agent"/>
+                <rng:ref name="name"/>
+              </rng:choice>
+            </rng:oneOrMore>
+            <rng:oneOrMore>
+              <rng:ref name="resp"/>
+            </rng:oneOrMore>
+          </rng:oneOrMore>
+        </rng:group>
+      </rng:choice>
```

Comment:
If I get it right the intention is not to predetermine the order of `resp`, `name`etc. Just out of curiosity:
Shouldnt' this do the same job?

```xml
<interleave>
    <oneOrMore>
       <ref name="bazga_resp"/>
    </oneOrMore>
    <oneOrMore>
       <choice>
          <ref name="bazga_name"/>
          <ref name="bazga_model.nameLike.agent"/>
       </choice>
    </oneOrMore>
</interleave>
         ```

Reply:
or in pure-ODD:
```xml
<sequence preserveOrder="false">
    <elementRef key="resp" minOccurs="1" maxOccurs="unbounded" />
    <alternate minOccurs="1" maxOccurs="unbounded">
        <elementRef key="name" />
        <classRef key="model.nameLike" />
    </alternate>
</sequence>
```